kolibri/plugins/learn/assets/src/views/ChannelRenderer/CustomContentRenderer.vue

```diff
@@ -120,6 +120,7 @@
             channel_id: this.topic.id,
             page: options.page ? options.page : 1,
             page_size: options.pageSize ? options.pageSize : 50,
+            only_topics: options.onlyTopics ? true : undefined,
```

Comment:
We can use the `kind` filter here, as I said above:

```suggestion
            kind: options.onlyTopics ? 'topic' : undefined,
```